pkg/aws/v1alpha3/awscontrolplane/validate_awscontrolplane.go

```diff
@@ -49,13 +49,27 @@ func NewValidator(config config.Config) (*Validator, error) {
 }
 
 func (v *Validator) Validate(request *admissionv1.AdmissionRequest) (bool, error) {
```

Comment:
This function was split into Create vs Update case because the namespace is only checked on creation.